public/app/features/panel/metrics_panel_ctrl.ts

```diff
@@ -199,6 +199,8 @@ class MetricsPanelCtrl extends PanelCtrl {
       interval: this.interval,
       intervalMs: this.intervalMs,
       targets: this.panel.targets,
+      panelRepeat: !!(this.panel.repeat || this.panel.repeatPanelId),
+      rowRepeat: !!(this.row.repeat || this.row.repeatRowId),
```

Comment:
this should not be required. you can look at scoped (repeated) variables in scopedVars